Remotion/Validation/UnitTests/Rules/ObjectValidationRuleTest.cs

```diff
@@ -1,21 +1,128 @@
-﻿using NUnit.Framework;
+﻿using System;
+using System.Collections.Generic;
+using Moq;
+using NUnit.Framework;
+using Remotion.Validation.Results;
+using Remotion.Validation.Rules;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Rules
 {
   [TestFixture]
   public class ObjectValidationRuleTest
   {
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void Validate ()
+    public void Validate_ReturnsValidationFailures ()
     {
+      Func<int, bool> condition = i => true;
+      var context = new ValidationContext(1);
```

Comment:
we can use an int here, some object would be more appropriate though for the intended semantics ob "Object Validation" since it refers to validating objects (entities), e.g. a Person.